recipes/parro/webview.js

```diff
@@ -0,0 +1,22 @@
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+const _path = _interopRequireDefault(require('path'));
+
+module.exports = Ferdium => {
+  function getMessages() {
+    let direct = 0;
+    let indirect = 0;
+    const FerdiumData = document.querySelector('#FerdiumMessages').dataset;
+    if (FerdiumData) {
+      direct = FerdiumData.direct;
+      indirect = FerdiumData.indirect;
+    }
+
+    Ferdium.setBadge(direct, indirect);
+  }
+
+  Ferdium.injectCSS(_path.join(__dirname, 'darkmode.css'));
```

Comment:
you should not remove the injection of the `service.css` - please resurrect it

Reply:
removed darkmode.css completly since there is none anymore.